.github/workflows/multichain-prod.yml

```diff
@@ -8,18 +8,13 @@ on:
           - mainnet
           - testnet
         description: mainnet or testnet network
+        default: testnet
         required: true
       image:
-        description: Full Artifact Registry image with tag (e.g. us-east1-docker.pkg.dev/pagoda-discovery-platform-prod/multichain/multichain-< testnet | mainnet >)
+        description: Full Artifact Registry image with tag (e.g. us-east1-docker.pkg.dev/pagoda-discovery-platform-prod/multichain-public/multichain-< testnet | mainnet >)
         required: true
-      tag:
-        description: Image tag that you wish to deploy, either by SHA or Version/latest
-      node_group:
-        type: choice
-        options:
-          - partner
-          - internal
-        description: Do you want to deploy the Parner nodes or internal Pagoda nodes?
+        default: us-east1-docker.pkg.dev/pagoda-discovery-platform-prod/multichain-public/multichain-testnet
```

Comment:
I think we also need to build it from the main branch instead of develop branch.Can we add that as well? Thanks

Reply:
You can select which branch to build from in the workflow dispatch run, it defaults to main branch.